src/main/kotlin/blackjack/domain/participant/State.kt

```diff
@@ -0,0 +1,51 @@
+package blackjack.domain.participant
+
+import blackjack.domain.card.Point
+
+sealed class State {
+    abstract fun compare(other: State): Match
+}
+
+object Hittable : State() {
+    override fun compare(other: State): Match {
+        throw IllegalStateException("Hittable 상태는 다른 상태와 비교될 수 없습니다")
+    }
+}
```

Comment:
```suggestion
sealed class State {
    abstract fun compare(other: State): Match

    object Hittable : State() {
        override fun compare(other: State): Match {
            throw IllegalStateException("Hittable 상태는 다른 상태와 비교될 수 없습니다")
        }
    }
}
```
대체로 sealed class는 위와 같이 State 클래스 내부로 묶습니다.
그리고 지금과 같이 클래스를 분리해서 사용하는 경우에는 주로 sealed interface를 활용합니다.